test/index.test.js

```diff
@@ -413,6 +413,31 @@ function describeXHRProps() {
     });
 
     describe('#responseType', function () {
+
+      it('supports binary data (arraybuffer responseType)', function (done) {
```

Comment:
Thanks for adding a test!

Reply:
let's say that at first i was so lost in here, that i wouldn't have been able to implement without it :)

Reply:
a test for json could be added as well